### PR TITLE
added integration tests

### PR DESCRIPTION
**Pull request by zvision**

### CODE REVIEW

File: src/it/java/se/knowit/bookitevent/ClientUtilIT.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        par.put("id", "72ab7c8b-c0d5-4ab2-8c63-5cf1ad0b439b");
+
+		Event eve = restTemplate.getForObject(url, Event.class, par);
+		System.out.println("Id:" + eve.getId() + " : " + eve.getName() + " : " + eve.getDescription());
+		Assert.hasText(Long.toString(eve.getId()),"1");
+		Assert.hasText(eve.getDescription(), "Ett himla bra event!");
+	}
+
+	@Test
+	public void getAllEventsDemo() {
```
Comment (ChristianVazques) [inline src/it/java/se/knowit/bookitevent/ClientUtilIT.java]: namn bör börja på test

File: src/it/java/se/knowit/bookitevent/ClientUtilIT.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		System.out.println("Id:" + eve.getId() + " : " + eve.getName() + " : " + eve.getDescription()));
+
+	}
+	
+	/*
+	public static void main(String args[]) {
+		ClientUtil util = new ClientUtil();
+		util.getAllEventsDemo();
+		util.getEventByIdDemo();
+	}*/
```
Comment (ChristianVazques) [inline src/it/java/se/knowit/bookitevent/ClientUtilIT.java]: Kan tas bort.

**Review by ChristianVazques — APPROVED** [review]

File: src/it/java/se/knowit/bookitevent/ClientUtilIT.java (hunk 1 above)
Comment (olamat) [reply src/it/java/se/knowit/bookitevent/ClientUtilIT.java]: There are many opinions on names of  tests :-)
https://dzone.com/articles/7-popular-unit-test-naming

File: src/it/java/se/knowit/bookitevent/ClientUtilIT.java (hunk 2 above)
Comment (olamat) [reply src/it/java/se/knowit/bookitevent/ClientUtilIT.java]: Agree. Don't keep out-commented code in VCS

**Review by olamat — CHANGES_REQUESTED** [review]
Remove the main method even if it has been commented

File: src/it/java/se/knowit/bookitevent/ClientUtilIT.java (hunk 2 above)
Comment (zvision) [reply src/it/java/se/knowit/bookitevent/ClientUtilIT.java]: Fixed  @olamat

**Review by olamat — APPROVED** [review]

**Pull request merged**